packages/paste-website/src/pages/components/pagination/index.mdx

```diff
@@ -0,0 +1,561 @@
+    }
+    mdx(fields: {slug: {eq: "/components/pagination/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+      }
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name={props.pageContext.frontmatter.title}
+  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
+  githubUrl="https://github.com/twilio-labs/paste/tree/main/packages/paste-core/components/pagination"
+  storybookUrl="/?path=/story/components-pagination--arrows"
+  abstractUrl="https://www.figma.com/file/E6KUvMhioUmAgN0nwmReTM/Paste-Components?node-id=1469%3A0"
+  data={props.data.allPasteComponent.edges}
+/>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+# Guidelines
+
+## About Pagination
+
+Use Pagination to split up content when a user’s goal is "to find a specific item [in a list] and click through to that destination page" (source: [Nielsen Norman Group](https://www.nngroup.com/articles/item-list-view-all/)). It’s used to communicate only where a user is in paged content. It isn’t used to show the status of each page, which is what a Progress Stepper ([coming soon](/roadmap)) would be used for.
+
+Pagination is often paired with [Tables](/components/table), one of the most commonly used components in Twilio and one of the primary ways customers view content in our products. Think of Pagination as an important navigation element that helps our customers understand exactly where they are and help them decide where to go next.
+
+### Accessibility Information
+
+- The Pagination component supports navigation by the use of either the Button or Anchor component. Use the Anchor component if the URL changes for each page. Use the Button component if the URL doesn’t change for each page. Check out our documentation on [when to use a Button vs. an Anchor](/components/button/#button-vs-anchor-link).
+- Use the Pagination component primarily when it controls a large part of the page in view, like a full-width Table. Since Pagination typically appears underneath the UI element it’s controlling, it needs to be obvious that it’s controlling the element above it, rather than anything else below it. For example, if you need to let users page through a small table in a card on a dashboard page, consider letting them expand the table and show Pagination on the expanded view instead.
+
+## Examples
+
+### Default pagination
+
+Use the default Pagination component for paged datasets that frequently increase in size, like event logs, or where the total number of pages is unknown.
+
+Whenever possible, show the _current_ page number, so that users know where they are in a dataset and get confirmation that they’ve moved to a new page.
+
+<LivePreview scope={{Pagination, PaginationItems, PaginationArrow, PaginationLabel}} noInline disabled>
+  {defaultExample}
+</LivePreview>
+
+#### Default pagination with Anchor
+
+Use the Anchor variant for Pagination if the URL changes for each page.
+
```

Comment:
About Button vs. Anchor: I think it would be useful if PaginationArrow and the props that determine whether it is a Button or Anchor are stated more clearly before the LivePreview here. I feel like it's especially important to do it since PaginationArrow defaults to Button, and I wasn't sure if the heading's mention of "Default" was about the Anchor/Button or Pagination itself.

Also, since there is a `variant` prop that is not about anchor vs. button, I think it'd be better to say something like "Use the Anchor **version** of Pagination Arrow by setting the `as="a"` prop and including an `href` prop."

Reply:
@gloriliale good suggestion. I added a little more to that section. Let me know if you think that works.